db/schema/patch-002-003.sql

```diff
@@ -0,0 +1,5 @@
+-- add locale to accounts table
+
+ALTER TABLE accounts ADD COLUMN locale VARCHAR(255);
+
+UPDATE dbMetadata SET value = '3' WHERE name = 'schema-patch-level';
```

Comment:
The patches are really just an example for usage and used only in dev. Production needs to create their own patches (but yes, we actually use these patches in prod). :)